src/drawer.py

```diff
@@ -163,3 +205,50 @@ def _draw_axis(self, axis: Axis):
         self.a.add_annotation('line', location,
                               Appearance(stroke_color=AXIS_COLOR,
                                          stroke_width=AXIS_STROKE_WIDTH))
+        
+    def _print_regression(self):
+        m, dm, n, dn = self.regression.m, self.regression.m_error, self.regression.n, self.regression.n_error
+
+        print()
+        print('Linear regression:')
+        print('y = m * x + n')
+        print('m  = {}'.format(m))
+        print('Δm = {}'.format(dm))
+        print('n  = {}'.format(n))
+        print('Δn = {}'.format(dn))
```

Comment:
where do I find this in the PAP script again?

Reply:
Page 10